data_contribution_2_data.md

```diff
@@ -0,0 +1,907 @@
+<!--
+
+author:   Arcus Library Sciences
+email: dlarcuslibraryscience@chop.edu
+version:  1.0.0
+current_version_description: Brief description of why this version exists
+module_type: standard
+docs_version: 1.1.0
+language: en
+narrator: UK English Female
+title: Arcus Project Template Orientation
+comment:  Arcus Archives serves as the canonical repository for research data at CHOP. Archiving data with Arcus safeguards and maintains research data in accordance with established archival protocols, also while promoting data sharing. The Data Contribution Module provides an overview of the reasons to archive research data with Arcus, as well as the processes involved in scoping data, conducting privacy assessments, and addressing technical requirements prior to data submission. This Module explores the Project Template directory structure, and delves into the specific steps required for archiving data within the Arcus platform.
```

Comment:
```suggestion
comment: Learn about how to disclose data to CHOP by making a data contribution!
long_description:  Arcus Archives serves as the canonical repository for research data at CHOP. Archiving data with Arcus safeguards and maintains research data in accordance with established archival protocols, also while promoting data sharing. The Data Contribution Module provides an overview of the reasons to archive research data with Arcus, as well as the processes involved in scoping data, conducting privacy assessments, and addressing technical requirements prior to data submission. This Module explores the Project Template directory structure, and delves into the specific steps required for archiving data within the Arcus platform.
```

Note that you're missing a "long_description" so I've taken the liberty of adding a comment and making your comment the long_description.  Feel free to tweak, this is just my suggestion.